recipes/funsor/LICENSE.md

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Made an upstream PR for this: https://github.com/pyro-ppl/funsor/pull/598

Reply:
thank you! 😀